### PR TITLE
py/java sol for cf-1514C

### PR DESCRIPTION
**Pull request by freakin23**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

### CODE REVIEW

**Comment by freakin23** [comment]
AC (JAVA): https://codeforces.com/problemset/submission/1514/279969501
AC (Python): https://codeforces.com/problemset/submission/1514/279968593

File: solutions/gold/cf-1514C.mdx [hunk 1]
```diff
@@ -75,4 +75,131 @@ int main() {
...
+			if (prefix_mod[i] == 1) { length_of_longest_subarray = i + 1; }
+		}
+
+		System.out.println(length_of_longest_subarray);
+
+		for (int i = 0; i < length_of_longest_subarray; i++) {
+			System.out.print(coprimes.get(i) + " ");
+		}
+		System.out.println();
+	}
```
Comment (SansPapyrus683) [inline solutions/gold/cf-1514C.mdx:136]: use `io.println` instead

File: solutions/gold/cf-1514C.mdx [hunk 2]
```diff
@@ -75,4 +75,131 @@ int main() {
...
+```py
+import math
+
+n = int(input())
+coprimes = []
+
+"""
+	This array stores all of the numbers
+	which are less than n and are coprime with n.
+"""
```
Comment (SansPapyrus683) [inline solutions/gold/cf-1514C.mdx:159]: sobbing what are these comment indentations

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: solutions/gold/cf-1514C.mdx (hunk 1 above)
Comment (freakin23) [reply solutions/gold/cf-1514C.mdx:136]: using io.println does't print anything

File: solutions/gold/cf-1514C.mdx (hunk 2 above)
Comment (freakin23) [reply solutions/gold/cf-1514C.mdx]: corrected these

File: solutions/gold/cf-1514C.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply solutions/gold/cf-1514C.mdx:136]: that’s because you didn’t put io.close at the end

File: solutions/gold/cf-1514C.mdx [hunk 3]
```diff
@@ -75,4 +75,128 @@ int main() {
...
+
+		/*
+		 * The longest subsequence whose product is 1 modulo n
+		 * is the longest subarray of coprimes with a product
+		 * of 1 modulo n.
+		 */
+
+		int number_of_coprimes = coprimes.size();
+		long[] prefix_mod = new long[number_of_coprimes];
+		prefix_mod[0] = 1;
```
Comment (SansPapyrus683) [inline solutions/gold/cf-1514C.mdx:108]: java uses camelcase

Comment (freakin23) [reply solutions/gold/cf-1514C.mdx]: done

**Review by SansPapyrus683 — APPROVED** [review]

**Pull request merged**